docs/tutorials/pre_executed/map_partitions.ipynb

```diff
@@ -54,24 +54,18 @@
    "cell_type": "code",
```

Comment:
<p>Are we doing<code> In this notebook you will learn...</code></p><p> </p><p>Maybe better <code>how to apply a user-defined-fuction</code> or <code>how to evaluate a user-define-function</code></p><p> </p><p>dataframe -&gt; pandas dataframe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/792/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Will update to use the new sections! Will add both "Learning Objectives" and "Introduction". And correct "dataframe".</p>

Reply:
<p>Per our discussion, will also make "when and how to use meta=" as a learning objective.</p>